doc/api/http.md

```diff
@@ -744,6 +744,35 @@ added: v0.4.0
 
 * {string} The request path.
 
+### `request.method`
+<!-- YAML
+added: v0.1.97
+-->
+
+* {string} The request method.
+
+### `request.host`
+<!-- YAML
+added: REPLACEME
+changes:
```

Comment:
The 'changes' section here and the one below should be removed.

Reply:
yes, I got it.